kubescape-operator.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: kubescape-operator
+  version: "0.2.92"
+  epoch: 0
+  description: Kubescape-Operator is an in-cluster component of the Kubescape security platform.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 81602c62580d67a78f30ff4670087392d39842c9
+      repository: https://github.com/kubescape/operator
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/go-viper/mapstructure/v2@v2.3.0
+        github.com/inspektor-gadget/inspektor-gadget@v0.40.0
+
+  - uses: go/build
+    with:
+      packages: .
+      output: operator
+
+update:
+  enabled: true
+  github:
+    identifier: kubescape/operator
+    strip-prefix: v
+
+test:
+  pipeline:
+    - runs: |
+        operator --help
+    - uses: test/kwok/cluster
+    - uses: test/daemon-check-output
+      with:
+        setup: |
+          kubectl config view --minify --raw > /tmp/kubeconfig.yaml
+
+          # Create required configuration directories
+          mkdir -p /etc/config /etc/credentials
+          echo "test-access-key" > /etc/credentials/access-key
+          echo "test-account-id" > /etc/credentials/account
+
+          # Creating STANDALONE configuration with empty service URLs
+          cat > /etc/config/clusterData.json << 'EOF'
+          {
+            "kubevulnURL": "",
+            "kubescapeURL": "",
+            "accountID": "test-account-id",
+            "clusterName": "test-cluster"
+          }
+          EOF
+
+          # Create minimal capabilities for standalone mode
+          cat > /etc/config/capabilities << 'EOF'
+          {
+            "capabilities": {
+              "configurationScan": "enable",
+              "vulnerabilityScan": "disable",
+              "nodeScan": "disable",
+              "continuousScan": "disable",
+              "runtimeObservability": "disable"
+            }
+          }
+          EOF
+
+          # Create minimal config for standalone
+          cat > /etc/config/config << 'EOF'
+          {
+            "namespace": "kubescape",
+            "port": "4002",
+            "triggersecurityframework": false,
+            "workerconcurrency": 1
+          }
+          EOF
+
+          kubectl create namespace kubescape
+        start: operator -kubeconfig /tmp/kubeconfig.yaml
+        timeout: 30
+        expected_output: |
+          Image version
+          credentials loaded
+        post: |
+          echo "Operator started successfully"
```

Comment:
this should be placed under `expected_output`. and lets have `wait-for-it` here to test port, and simple curl request if possible

Reply:
added

Reply:
https://github.com/wolfi-dev/os/pull/57973/commits/fb810b1955ed2f87628f9da61a12cf521e4c0327